src/explore-education-statistics-frontend/src/modules/find-statistics/components/ReleaseTypeFilters.tsx

```diff
@@ -0,0 +1,59 @@
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import FormRadioGroup from '@common/components/form/FormRadioGroup';
+import useToggle from '@common/hooks/useToggle';
+import { releaseTypes, ReleaseType } from '@common/services/types/releaseType';
+import { ReleaseTypesModal } from '@frontend/modules/find-statistics/components/FilterModals';
+import { FilterChangeHandler } from '@frontend/modules/find-statistics/components/Filters';
+import React from 'react';
+
+interface Props {
+  releaseType?: ReleaseType | 'all';
+  onChange: FilterChangeHandler;
+}
+
+const ReleaseTypeFilters = ({
+  releaseType: initialReleaseType = 'all',
+  onChange,
+}: Props) => {
+  const [showModal, toggleModal] = useToggle(false);
+  const releaseTypeOptions = [
+    { label: 'Show all', value: 'all' },
+    ...Object.keys(releaseTypes).map(type => ({
+      label: releaseTypes[type as keyof typeof releaseTypes],
+      value: type,
+    })),
+  ];
+
+  return (
+    <form id="releaseTypeFilters">
+      <FormRadioGroup
+        hint={
+          <ButtonText onClick={toggleModal.on}>
+            What are release types?
+          </ButtonText>
+        }
+        id="releaseType"
+        legend="Filter by release type"
```

Comment:
The UI is a little busy with headings. Would suggest we use `legendHidden` to hide this. We would get this:

![image](https://user-images.githubusercontent.com/9917868/205950611-d4cb5ad9-f865-4c76-8cb6-5e4c61b02036.png)